/**:
  ros__parameters:

    robot_width: 0.8
    turn_cost: 0
    min_weight: 25
    update_time_s: 0.5

    avoidance_zone_corner_x: 3.88
    avoidance_zone_corner_y: 3.00

    arena_width: 6.88
    arena_height: 5.00

    output_frame: "world"
